src/PresentationalComponents/ComplianceScore/ComplianceScore.test.js

```diff
@@ -10,8 +13,9 @@ describe('auxiliary functions to reducer', () => {
       compliant: false,
     };
 
-    const dangerIcon = mount(<ComplianceScore {...system} />);
-    expect(toJson(dangerIcon)).toMatchSnapshot();
+    const { container } = render(<ComplianceScore {...system} />);
+
+    expect(queryByText(container, '10%')).not.toBeNull();
```

Comment:
nit-pick:
```suggestion
    expect(screen.getByText('10%'));
```
should be almost an equal change since `get` queries will error if there are no elements or more than one element found.

Also works for many other places where you import `queryByText` from @testing-library/dom (will not comment on those).

Reply:
Done.